src/main/drivers/bus_spi_ll.c

```diff
@@ -518,17 +548,17 @@ void spiSequence(const extDevice_t *dev, busSegment_t *segments)
             dmaSafe = false;
             break;
         }
-#elif defined(STM32F4)
+#elif defined(STM32G4)
         // Check if RX data can be DMAed
         if ((checkSegment->rxData) &&
-            // DTCM can't be accessed by DMA1/2 on the F4
-            (IS_DTCM(checkSegment->rxData)) {
+            // CCM can't be accessed by DMA1/2 on the G4
+            IS_CCM(checkSegment->rxData)) {
             dmaSafe = false;
             break;
         }
         if ((checkSegment->txData) &&
-            // DTCM can't be accessed by DMA1/2 on the F4
-            (IS_DTCM(checkSegment->txData)) {
+            // CCM can't be accessed by DMA1/2 on the G4
+            IS_CCM(checkSegment->txData)) {
```

Comment:
And it seems that you are missing a '(' before the `IS_CCM()`.

Reply:
This was actually an error in the previous code which was never included as this file was never built for the F4.

```
        // Check if RX data can be DMAed
        if ((checkSegment->rxData) &&
            // CCM can't be accessed by DMA1/2 on the G4
            IS_CCM(checkSegment->rxData)) {
            dmaSafe = false;
            break;
        }
```